spec/unit/mutant/context_spec.rb

```diff
@@ -31,27 +31,5 @@
 
       it { should eql(expected) }
     end
-
-    context 'with Class as scope' do
```

Comment:
Without this removal there is a un-reachable mutation covered branch in code that was selected via `--since` for this commit.